packages/ember-metal/lib/deprecate_property.js

```diff
@@ -21,10 +21,6 @@ import { set } from 'ember-metal/property_set';
 */
 
 export function deprecateProperty(object, deprecatedKey, newKey, options) {
-  function deprecate() {
```

Comment:
This is also used by the `deprecateProperty` setter below (they share the same deprecation call).  The setter below now calls the imported `deprecate()`, but doesn't actually work..